pkg/image/admission/imagepolicy/api/types.go

```diff
@@ -13,18 +13,34 @@ const IgnorePolicyRulesAnnotation = "alpha.image.policy.openshift.io/ignore-rule
 type ImagePolicyConfig struct {
 	unversioned.TypeMeta
 
+	// ResolveImages indicates what kind of image resolution should be done.  If a rewriting policy is chosen,
+	// then the image pull specs will be updated.
+	ResolveImages ImageResolutionType
```

Comment:
I think I'm fine with this.  Since only things that are either in the integrated registry or have a digest can be resolved, it's not like this takes options away.